src/core/synchronization/mod.rs

```diff
@@ -397,7 +398,7 @@ impl PoseEstimator {
         find_offset::find_offsets(ranges, &gyro, initial_offset, search_size, params, progress_cb, cancel_flag)
     }
 
-    pub fn find_offsets_rssync<F: Fn(f64) + Sync>(&self, ranges: &[(i64, i64)], initial_offset: f64, search_size: f64, params: &ComputeParams, progress_cb: F, cancel_flag: Arc<AtomicBool>) -> Vec<(f64, f64, f64)> { // Vec<(timestamp, offset, cost)>
+    fn collect_points_for_rssync(&self,  ranges: &[(i64, i64)]) -> Vec<Vec<((i64, OpticalFlowPoints), (i64, OpticalFlowPoints))>> {

```

Comment:
We can maybe move this function to `FindOffsetsRssync`?

Reply:
I do not have any ideas how to do this without depending on PoseEstimator and FrameResult in find_offset_rssync

Reply:
ok, it can stay then